src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TransformToVisual.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading.Tasks;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MUXControlsTestApp.Utilities;
+using Private.Infrastructure;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls
+{
+	[TestClass]
+	public class Given_TransformToVisual
```

Comment:
The tests for the "TransfromToVisual` are in the `UIElementTests` class https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/UIElementTests/UIElementTests.cs#L20